src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Dsa.cs

```diff
@@ -65,7 +65,7 @@ internal static int DsaKeySize(SafeDsaHandle dsa)
             // there were some leading zero bytes truncated due to BIGNUM semantics.
             // This minimizes the change the key size is wrong due to leading zero bits; there is only
             // a 2^64 chance that the key has 64 leading zero bits and thus the keysize will be wrong.
```

Comment:
I'm assuming for each bit in a given p there is a 50% chance of it being zero. So for example if the first 8 bits are zero, there is one less byte required to store p in a bignum due to the semantics of bignum which doesn't pad leading bytes (thus the requested keysize would be off by 8 bits in this case). So 2^64 comes into place because we round 8 bytes (64 bits), thus the key size would be wrong (off by 64) if the first 64 bits are all zero.